libimgui-docking/manifest

```diff
@@ -0,0 +1,20 @@
+: 1
+name: libimgui-docking
+version: 1.90.8
+project: imgui
+summary: Dear ImGui: Bloat-free Graphical User interface for C++ with minimal dependencies
```

Comment:
Optional:
You could add some notes here that this package has docking support. Any thoughts on deviating from the summary of the upstream repo @boris-kolpackov ?

Reply:
Will wait for Boris though on the matter before addressing👍

Reply:
We routinely editorialize upstream description (e.g., to filter out weasel words), so go for it.

Here are the guidelines: https://build2.org/build2-toolchain/doc/build2-toolchain-packaging.xhtml#core-root-manifest-summary